tests/python/RadonBearTest.py

```diff
@@ -1,39 +1,65 @@
-from bears.python.RadonBear import RadonBear
-from coalib.testing.LocalBearTestHelper import verify_local_bear
-
-test_file1 = """
-def simple():
-    pass
-"""
+from queue import Queue
 
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.settings.Section import Section
+from coalib.settings.Setting import Setting
+from bears.python.RadonBear import RadonBear
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
 
-test_file2 = """
-class class1():
-    pass
-"""
-
-test_file3 = 'def f():\n' + ('    assert True\n' * 50)
 
+class RadonBearTest(LocalBearTestHelper):
+    def setUp(self):
+        self.section = Section('test')
+        self.uut = RadonBear(self.section, Queue())
+        self.test_file1 = ['def simple():\n',
```

Comment:
the unmodified test files and old classes can stay as module variables.

But add "Rank" somewhere in the old class names.

Then the class exists for the new functionality, and the old classes will be deleted in one release.

Reply:
remove these class variables.
they are not needed.

use the module variables.